### ISSUE TITLE
ZeroDivisionError in gv.sdev for non-gvar arrays w/ length > 50

### ISSUE THREAD

**Issue by dchackett** [gplepage/gvar#34]
`gv.sdev` throws a ZeroDivisionError when applied to arrays or lists of pure numbers (i.e. not gvars) longer than 50 elements. The values don't seem to matter.

To reproduce: 
```
gv.sdev(np.arange(50)) # == np.zeros(50)
gv.sdev(np.arange(51)) # ZeroDivisionError
gv.sdev([0]*51) # ZeroDivisionError
gv.sdev(np.random.rand(51)) # ZeroDivisionError
```

**Comment by gplepage** [comment]
What version of ``gvar`` are you using? Also which operating system?

This problem should have been fixed in v11.11, in response to the same issue being raised last August.

**Comment by dchackett** [comment]
Oops; that was for 11.10, can confirm the issue is resolved by updating to 11.11. Apologies for the confusion, I thought I had checked in the latest version, but had my conda environments mixed up.

**Comment by gplepage** [comment]
No problem.
